cloud/aws/deploytf/.nitric/modules/stack/main.tf

```diff
@@ -5,15 +5,15 @@ resource "random_string" "id" {
 }
 
 resource "aws_resourcegroups_group" "group" {
-  name = "nitric-resource-group-${random_string.id.result}"
+  name = "${var.project_name}-${var.stack_name}-${random_string.id.result}"
```

Comment:
Will this force a recreation of all resources in an existing user's AWS project?

Reply:
This will only replace the resource group. Other resources won't change.